Samples/IonicCapacitorAngularV2/ct-demo-ionic-angular-cap/src/app/home/home.page.ts

```
@@ -0,0 +1,266 @@
+  onItemClick( item: string) {
+    this.presentToast(item);
+    switch (item) {
+      case "Record Event":
+        CleverTap.recordEventWithName("foo");
+        break;
+      case "Record event with properties":
+        const map = {
+          bar: 'zoo',
+          startdate: Date(),
+          enddate: Date(),
+        };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)